packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -3623,6 +3623,93 @@ void main() {
     );
   });
 
+  testWidgets(
+    'Can drag the left handle while the right handle disappears',
+    (WidgetTester tester) async {
+      // text is longer than textfield width.
```

Comment:
Capital letter and period for this and all of the comments below.

Reply:
fixed all